rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+
+  def initialize(date)
+    @date = date
+
+    @person = person
+    person.add_rental(self)
+
+    @book = book
```

Comment:
## OPTIONAL 
*  Getters and Setters for @book and @person instance variables should be present 
* `The constructor of Rental should be modified so Book and Person are set in it to break the many-to-many relationship.`  [Link to example article ](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`)